projects/vdk-plugins/vdk-jobs-troubleshooting/README.md

```diff
@@ -1 +1,245 @@
-### VDK-JOBS-TROUBLESHOOTING Plugin
+## VDK-JOBS-TROUBLESHOOTING Plugin
+
+The VDK JOB Troubleshooting plugin provides the ability to add various troubleshooting utilities which can be accessed
+during the data job runtime.
+
+Generally it's quite hard to produce a thread dump of a python process running inside a kubernetes pod. So this plugin
+provides a thread-dump utility
+
+### Adding the plugin to your data job
+
+Before you can use the plugin you should add it to your data job or custom-sdk, by adding the following line to your
+requirements.txt file:
+
+```commandline
+install_requires = vdk-jobs-troubleshooting
+```
+
+Next you need to add the list of utilities to use to your data job configuration. For example, to enable the thread-dump
```

Comment:
You mean the `config.ini` file? If so, maybe add the title section where the configuration needs to go, as to avoid cryptic errors, caused by placing it in the wrong place and vdk not picking it.
```ini
[vdk]

TROUBLESHOOT_UTILITIES_TO_USE=thread-dump
```

Reply:
ok